docs/source/reference/_create_conda_env.rst

```diff
@@ -0,0 +1,383 @@
+    <pre>
+    $ <b>conda config --show solver</b>
+    </pre>
+    </ul>
+
+    </details>
+
+
+
+
+Install the bluesky environment
+-------------------------------
+
+The following commands install the ``bluesky_2023_2`` environment inside the
+``bluesky`` directory  that was created when installing a new bluesky instrument
+(see `setup a bluesky instrument
+<https://bcda-aps.github.io/bluesky_training/instrument/_install_new_instrument.html#setup-a-bluesky-instrument>`__).
+
+
+
+Note that the installation takes several minutes.
+
+.. raw:: html
+
+    <pre>
+    $ <b>cd ~/bluesky</b>
+    $ <b>conda env create \
+        --force \
+        -n bluesky_2023_2 \
+        -f ./environments/environment_2023_2.yml \
+        --solver=libmamba</b>
+    </pre>
+
+    <details><summary>Details</summary>
+    
+    <ul> 
+    <li>The <code>--force</code> option will replace any existing environment by this
+    name without asking for confirmation. Remove this option if you wish.</li>
+    <li>The <code>-n bluesky_2023_2</code> sets the name of the conda environment to be
+    created.</li> 
+    <li>The <code>-f ./environments/environment_2023_2.yml</code> option names the 
+    YAML file to be used. We create different versions of the YAML file, named for the APS operating cycle (2021-1, 2023-2, …), as the suite of packages for a working installation may change over time. By keeping all these files in the environments subdirectory, we can restore any of these environments with a simple command.</li> 
+    <li>The <code>--solver=libmamba</code> option will use the conda-libmamba-solver. This option is optional (can be removed) but result in a much faster installation of the bluesky environment. The libmanba installation is described in the previous section.</li> 
+    </ul>
+
+    </details>
+
+
+
+Once finished, the installer will report the commands to manage the new environment:
+
+.. raw:: html
+
+    <pre>
+    #
+    # To activate this environment, use
+    #
+    #     $ conda activate bluesky_2023_2
+    #
+    # To deactivate an active environment, use
+    #
+    #     $ conda deactivate
+    </pre>
+
+
+
+
+
+Create an alias to activate the bluesky environment
+---------------------------------------------------
+
+Creating a bash alias is like creating a custom shortcut. You can do this by
+editing the ``~/.bashrc`` and  ``~/.bash_aliases`` files, which are
+configuration files for your bash shell. Here's a simple step-by-step guide:
+
+.. raw:: html
+
+   <ol>
+   <li>Open a terminal.</li>
+   <li>Open the <code>~/.bashrc</code> and <code>~/.bash_aliases</code> files with your prefered text editor, 
+   <i>e.g.</i>:
+   <pre>
+   $ <b> gedit ~/.bashrc ~/.bash_aliases </b>
+   </pre>
+   If any of those files do not exist, this command will create blank ones. 
+   </li>
+   <li> In <code>~/.bashrc</code>, scroll down to the end of the file or find 
+   a suitable place to add the following lines:
```

Comment:
FYI: this may already be present with new user accounts.  Here is the part I found in `~/.bashrc` with my recent new OS installation:

```bash
# Alias definitions.
# You may want to put all your additions into a separate file like
# ~/.bash_aliases, instead of adding them here directly.
# See /usr/share/doc/bash-doc/examples in the bash-doc package.

if [ -f ~/.bash_aliases ]; then
    . ~/.bash_aliases
fi
```

The `.` is the same as `source`.  This block checks if the file exists before trying to source it.